tapsend/allocation.go

```diff
@@ -152,6 +152,9 @@ type Allocation struct {
 	// as the relative time lock on the virtual output.
 	Sequence uint32
 
+	// LockTime is the actual CLTV value that will be set on the output.
+	LockTime uint64
```

Comment:
Perhaps rename the `CLTV` field above to make it clearer that it's primarily there for HTLC sorting tie breakers? Otherwise could be confusing which of the two fields to set, and if they're actually the same value or not.